src/main/java/build/buildfarm/common/config/BuildfarmConfigs.java

```diff
@@ -36,7 +36,7 @@ public static BuildfarmConfigs getInstance() {
     if (buildfarmConfigs == null) {
       buildfarmConfigs = new BuildfarmConfigs();
       try {
-        loadDefaultConfigs("examples/config.yml");
+        loadDefaultConfigs("../build_buildfarm/examples/config.yml");
```

Comment:
wondering if we want to make the path an argument to the program so that people wont have to change it from time to time.

Reply:
This is for default path to the example config file which shouldn't change. The base math worked fine when building directly, but broke when I was trying to build and test as a dependency of anothe repo as bazel was attempting to load it from my custom named workspace and not the workspace of build_buildfarm. This corrects it for this case. Not sure if there is a better way to do it and if so I'd like to get it fixed properly vs this solution which seems a bit hacky.